airflow/configuration.py

```diff
@@ -608,6 +609,48 @@ def getjson(self, section, key, fallback=_UNSET, **kwargs) -> Union[dict, list,
         except JSONDecodeError as e:
             raise AirflowConfigException(f'Unable to parse [{section}] {key!r} as valid json') from e
 
+    def gettimedelta(self, section, key, fallback=None, **kwargs) -> Optional[datetime.timedelta]:
```

Comment:
nit, wondering if we can call it `get_timedelta` given there are two `t`s.

Reply:
hi, this method aims to provide a simple functionality to take a string value from the config file, then try to convert it into an integer, and pass it into timedelta in seconds. So, it is expected to be returned as _a single timedelta object_. It is very similar to the other helper methods like _getint_, _getfloat_, _getboolean_, etc. So I think the method does not support the operation for multiple `t`s.

If I misunderstood your comment, I would like to hear more on this to understand the use cases.

Reply:
hi @sagmansercan i was thinking about the naming of this method, `gettimedelta` vs `get_timedelta`.

`gettimedelta` conforms the current naming convention under `conf`, like `getint`. However, there are two `t`s in the naming, making it harder to read. but it is not a big deal

Reply:
Oh, now I see your point :) Yes, it may be a little bit annoying, but as you said, it is compatible with the naming convention in the `configuration.py `. I can apply renaming If everyone is okay with _snake_case_, but for now I am leaving it as it is